### ISSUE TITLE
Force check on Background not working

### ISSUE THREAD

**Issue by joseashb** [airsquared/blobsaver#470]
**Describe the bug**
Unable to force start background check on 3.2.1

**Screenshots**
![image](https://user-images.githubusercontent.com/47283948/186252479-805f58ab-2171-4bb6-8f6d-f5fab3485c4d.png)

**Desktop:**
 - OS Version: Windows 10 LTSC 2019
 - iOS Device: Multiple

**Checklist (place an x between the brackets to mark as completed):**
 - [x] I have made sure I am on the latest version of blobsaver
 - [x] I have checked other open issues to see if my problem has already been reported
 - [x] I have tried common troubleshooting tactics such as restarting my computer and reinstalling blobsaver

**Log(s)**
```
ERROR: Invalid argument/option - '\airsquared\blobsaver\BackgroundService'.
Type "SCHTASKS /RUN /?" for usage.
```

**Comment by Infinitay** [comment]
I'm receiving this same issue when force checking. I hope this doesn't mean that my background service was not running at all. Although it seems to have still worked because I have local blobs for iOS 16.

```
ERROR: Invalid argument/option - '\airsquared\blobsaver\BackgroundService'.
Type "SCHTASKS /RUN /?" for usage.
```

**Comment by airsquared** [comment]
Please run the following command in cmd:

    schticks /run /tn \airsquared\blobsaver\BackgroundService

and paste the output here.

**Comment by airsquared** [comment]
Oops yeah I did mean `schtasks`. Thanks, that confirms that background saving is working, but "Force check" was broken on Windows. Will be fixed in the next version.

**Comment by airsquared** [comment]
Fixed in v3.5.0.
